lib/views/general/_responsive_footer.html.erb

```diff
@@ -24,7 +24,7 @@
         </li>
         <li>
           <h4><%= _("A project by") %></h4>
-          <%= link_to image_tag('logo.png', :alt => "Diritto di Sapere logo"), "http://www.dirittodisapere.it/", :class => 'footer-logo' %>
+          <%= link_to image_tag('dds-logo.png', :alt => "Diritto di Sapere logo"), "http://www.dirittodisapere.it/", :class => 'footer-logo' %>
```

Comment:
Change requested in https://github.com/mysociety/dirittodisapere-theme/issues/92#issuecomment-223601584